element_miniscope/miniscope.py

```diff
@@ -233,19 +238,19 @@ def make(self, key):
             with open(recording_metadata.as_posix()) as f:
                 metadata = json.loads(f.read())
 
-            with open(recording_timestamps, newline= '') as f:
+            with open(recording_timestamps, newline='') as f:
                 time_stamps = list(csv.reader(f, delimiter=','))
 
-            nchannels = 1 # Assumes a single channel
+            nchannels = 1  # Assumes a single channel
             nframes = len(time_stamps)-1
             px_height = metadata['ROI']['height']
             px_width = metadata['ROI']['width']
-            fps = int(metadata['frameRate'].replace('FPS',''))
+            fps = int(metadata['frameRate'].replace('FPS', ''))
             gain = metadata['gain']
-            spatial_downsample = 1 # Assumes no spatial downsampling
+            spatial_downsample = 1  # Assumes no spatial downsampling
             led_power = metadata['led0']
             time_stamps = np.array([list(map(int, time_stamps[i]))
-                                       for i in range(1,len(time_stamps))])
+                                    for i in range(1, len(time_stamps))])
```

Comment:
Does this really intend to skip `time_stamps[0]`?

Reply:
Thanks @dimitri-yatsenko.  This is implemented to skip the heading of the CSV file:
![image](https://user-images.githubusercontent.com/871137/167035229-7d4b0f85-fb44-423d-b331-3c0e97b784c0.png)